### ISSUE TITLE
Could  you tel me the best  “Training Loss ”and “slim acc ” in the training iteration?

### ISSUE THREAD

**Issue by zhaopanpan73** [xuehy/Cross-View-Gait-Deep-CNNs#2]
I have run the code .but the  best  “slim acc ” is just 0.85。I feel confused about this

**Comment by xuehy** [comment]
What do you mean by "slim acc"? How many iterations do you run?

**Comment by zhaopanpan73** [comment]
“”slim acc“ is in the train.py “ print('{}, iter = {}, loss = {}, sim acc = {}'.format(localtime, iteration, loss.cpu().item(), acc/n))” 。iteration=440000 .now is running on 140000 iteration

**Comment by zhaopanpan73** [comment]
I am wondering to stop the train code.because I don't know  whether the network is convergenced

**Comment by xuehy** [comment]
You need more iterations. 
![newplot](https://user-images.githubusercontent.com/2055701/41271172-a337719c-6e41-11e8-8d43-2d28756839bc.png)
![result1](https://user-images.githubusercontent.com/2055701/41271173-a4ddbbaa-6e41-11e8-8cb4-84a1a6adccb1.png)

After 45k iterations, I can reach the accuracy of 92.8%.

**Comment by zhaopanpan73** [comment]
![1897112491](https://user-images.githubusercontent.com/31726025/41273625-ca8c356a-6e4c-11e8-94cc-dc01a9d73222.jpg)
Maybe ,you are right. here is my training situation .According to the paper.  I select the first 24 subjects to training the network. the next 24 subjects for validation.and the left 76 subjects for test .it's a little different from your experiment. So after 140k iteration .the network still misconvergence
